packages/neos-ui-editors/src/Editors/Image/index.js

```diff
@@ -210,10 +210,14 @@ export default class ImageEditor extends Component {
     }
 
     handleChooseFromMedia = () => {
-        const {secondaryEditorsRegistry} = this.props;
+        console.log('options (r)', this.props.options)
+        const {secondaryEditorsRegistry, options} = this.props;
         const {component: MediaSelectionScreen} = secondaryEditorsRegistry.get('Neos.Neos/Inspector/Secondary/Editors/MediaSelectionScreen');
 
-        this.props.renderSecondaryInspector('IMAGE_SELECT_MEDIA', () => <MediaSelectionScreen type="images" onComplete={this.handleMediaSelected}/>);
```

Comment:
This would be possibly breaking if someone made a custom selection screen and relied on `type`.
Maybe keep it and mark deprecated for 6.0?

Reply:
Good call, I added the type prop again and turn it into `constraint.mediaTypes=['image/*']` if set to "images"